maps/away/away_site/tajara/peoples_station/peoples_station.dmm

```diff
@@ -328,11 +328,16 @@
 	},
 /area/peoples_station/training)
 "bA" = (
+/obj/effect/map_effect/window_spawner/full/shuttle{
+	spawn_firedoor = 1;
+	spawn_grille = 1;
+	color = "#6C7364";
+	frame_color = "#6C7364"
+	},
 /obj/machinery/door/blast/regular/open{
-	id = "peoples_station_transport_blast";
+	id = "elyran_shuttle_shutters";
```

Comment:
?

Reply:
huh. is that still on there? i'll fix it soon